perfmetrics/scripts/continuous_test/gcp_ubuntu/build.sh

```diff
@@ -36,16 +35,26 @@ MOUNT_POINT=gcs
 # The VM will itself exit if the gcsfuse mount fails.
 gcsfuse $GCSFUSE_FLAGS $BUCKET_NAME $MOUNT_POINT
 
-# Executing perf tests
-chmod +x run_load_test_and_fetch_metrics.sh
-./run_load_test_and_fetch_metrics.sh
+chmod +x perfmetrics/scripts/run_load_test_and_fetch_metrics.sh
+./perfmetrics/scripts/run_load_test_and_fetch_metrics.sh
 # Copying gcsfuse logs to bucket
 gsutil -m cp $LOG_FILE gs://gcs-fuse-dashboard-fio/fio-gcsfuse-logs/
 
 # Deleting logs older than 10 days
-python3 utils/metrics_util.py gcs/fio-gcsfuse-logs/ 10
+python3 perfmetrics/scripts/utils/metrics_util.py gcs/fio-gcsfuse-logs/ 10
 
 # ls_metrics test
-cd "./ls_metrics"
-chmod +x run_ls_benchmark.sh
-./run_ls_benchmark.sh
+chmod +x perfmetrics/scripts/ls_metrics/run_ls_benchmark.sh
+./perfmetrics/scripts/ls_metrics/run_ls_benchmark.sh
+sudo umount gcs
+
+# Executing implicitdir integration tests for the testbucket flag
+GODEBUG=asyncpreemptoff=1 go test ./tools/integration_tests/implicitdir/ -v --testbucket=gcsfuse-implicitdir-tests
+# Mounting gcsfuse for running integration tests for the mountedDirectory flag
+BUCKET_NAME=gcsfuse-implicitdir-tests
+MOUNT_POINT=gcs
+# The VM will itself exit if the gcsfuse mount fails.
+gcsfuse $GCSFUSE_FLAGS $BUCKET_NAME $MOUNT_POINT
+# Executing implicitdir integration tests for the mountedDirectory flag
+GODEBUG=asyncpreemptoff=1 go test ./tools/integration_tests/implicitdir/ -v --mountedDirectory=../../../$MOUNT_POINT
```

Comment:
why are we doing so many changes just to run integration tests. why can't it be just cd to home folder and then run integration tests from there?

Reply:
So currently we will be in ls_metrics folder.
you mean to do

cd ../../../.
GODEBUG=asyncpreemptoff=1 go test ./tools/integration_tests/implicitdir/ -v --testbucket=gcsfuse-implicitdir-tests

this?

Reply:
I thought it will increase confusion to track current folder. and as we did it same for perf_tests we should follow the same.

Reply:
i dont understand. everytime somthing is changing in this file , we are updating the folder which requires changes in multiple places. i dont prefer doing it always. it might break anything.
just do a cd at the end and run integration tests

just "cd" should also be sufficient. add a comment why are you doing this.